.prod/nginx.conf

```diff
@@ -1,22 +1,39 @@
-server {
-    listen       80;
-    server_name  localhost;
+pid        /tmp/nginx.pid; #for running as non-root 
 
-    location /healthz {
-        return 200 'OK';
-    }
+events {
+    worker_connections  4096;  ## Default: 1024
```

Comment:
What is the purpose of this change? Looks benign, but would be nice to know why.

Reply:
Probably came from a config template for high traffic websites: https://stackoverflow.com/questions/56723449/nginx-error-1024-worker-connections-are-not-enough

Reply:
Does this need to be changed or can it be left as is?

Reply:
Can be left as is.